AUTHORS

```diff
@@ -0,0 +1,4 @@
+Florent Revest revestflo@gmail.com
+Doug Koellmer dougkoellmer@hotmail.com
+Justus Tartz git@jrtberlin.de
+Doomsdayrs doomsdayrs@gmail.com
```

Comment:
This line doesn't end with a newline character.

Reply:
That shall be fixed